string ns_goal
---
bool success